.github/workflows/node-ci.yml

```diff
@@ -9,10 +9,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@master
-    - name: Set up Node.js 14.21.3
+    - name: Set up Node.js 22.1.0
       uses: actions/setup-node@v1
       with:
-        node-version: 14.21.3
+        node-version: 22.1.0
```

Comment:
I wanted to test the new node version by building and running p2a on my workstation, but with both node version 14 and node version 22, I got the runtime error `TypeError: item is undefined`. Do you know offhand what I'm missing (probably an environment variable)?

Reply:
I think the runtime error is because the `MODEL_ID` environment variable has changed and fetchSummayStatistics is using the old default 'anusplin' model and climatology. This is updated in the Part 4 PR.  https://github.com/pacificclimate/plan2adapt-v2/blob/7b4b7c4b9e1f64c9e89aad27bc06138df6706e79/src/data-services/summary-stats.js#L26-L27